test/TestProjects/PetStore-TypeSpec/PetStore-TypeSpec.tsp

```diff
@@ -99,32 +99,6 @@ model Tuna extends Fish {
 @summary("Manage your pets.")
 @route("/pets")
 namespace Pets {
-  @doc("Delete a pet.")
-  @summary("delete.")
-  @delete
-  @convenientAPI(true)
-  op delete(...PetId): Response<200> | Error;
-
-  // @fancyDoc("List pets.")
-  // op list(@query nextLink?: string): OkResponse<ResponsePage<Pet>> | Error;
-
-  @doc("Returns a pet. Supports eTags.")
-  @convenientAPI(true)
-  op read(...PetId): Pet | (NotModifiedResponse & Pet) | Error;
-
-  @convenientAPI(true)
-  @post op create(@body pet: Pet): Pet | Error;
-
-  @convenientAPI(true)
-  @get op getPetByKind(@path kind: PetKind): Pet | Error;
-
-  @convenientAPI(true)
-  @get op GetFirstPet(@query start?: int32): Pet | Error;
-
-  @convenientAPI(true)
-  @route("/getFish")
-  @get op getFish(@query kind?: string): Fish | Error;
-
```

Comment:
Ask @archerzz if we still need LRO tests in the autorest.csharp. If not, we could remove the whole file.

Reply:
> Ask @archerzz if we still need LRO tests in the autorest.csharp. If not, we could remove the whole file.

Let's focus on `cadl-ranch` LRO tests for now. We should guide green field SDKs to use standard template as much as possible.

Reply:
fixed